src/lib/viewers/controls/annotations/AnnotationsTargetedTooltip.tsx

```diff
@@ -44,6 +45,10 @@ function AnnotationsTargetedTooltip({ children, isEnabled = false }: Props): JSX
                         experiences.tooltipFlowAnnotationsExperience.onClose();
                         setIsForced(false);
                     },
+                    onComplete: (): void => {
+                        experiences.tooltipFlowAnnotationsExperience.onComplete();
+                        setShouldTargetAnnotationsTooltip(false);
```

Comment:
I'm a little confused as to how this is different than our `shouldTarget` variable above. Does calling `onComplete()` not set `canShow` to false?

Reply:
That is correct, calling onComplete does not set canShow to false.